src/Services/Ordering/Ordering.API/Application/Behaviors/LoggingBehavior.cs

```diff
@@ -1,23 +1,16 @@
-﻿using MediatR;
-using Microsoft.eShopOnContainers.BuildingBlocks.EventBus.Extensions;
-using Microsoft.Extensions.Logging;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Ordering.API.Application.Behaviors
+﻿namespace Microsoft.eShopOnContainers.Services.Ordering.API.Application.Behaviors;
```

Comment:
```suggestion
﻿namespace Microsoft.eShopOnContainers.Services.Ordering.API.Application.Behaviors;
﻿
```

I'm noticing that while we're now using file-scoped namespaces, that the namespace itself changed - why is that?